usecase/providerd.go

```diff
@@ -133,6 +135,7 @@ func (g *providerDaemon) Start(ctx context.Context) <-chan []error {
 func newAuthorizationd(cfg config.Config) (service.Authorizationd, error) {
 	return providerd.New(
 		providerd.WithAthenzURL(cfg.Athenz.URL),
+func newAuthzD(cfg config.Config) (service.Authorizationd, error) {
```

Comment:
why shorten?

Reply:
go prefer shorter name.
minor change only, you can just revert it.